### PR TITLE
added HW5

### PR DESCRIPTION
**Pull request by Vladislav-Panchehin**

### CODE REVIEW

File: VladislavPanchehin/HW4/5.1.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿
+intenger_list = [1, 2, 3, 4, 5]
+
+# variant 1
+
+for digit in intenger_list:
+    print(float(digit))
+
+# variant 2
+
```
Comment (kolyasalubov) [inline VladislavPanchehin/HW4/5.1.py:10]: excellent implementation

File: VladislavPanchehin/HW4/5.2.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿# F(n)=F(n−1)+F(n−2)
+
```
Comment (kolyasalubov) [inline VladislavPanchehin/HW4/5.2.py:2]: good job

File: VladislavPanchehin/HW5/5.3.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# variant 1
+# num = input("Entered your number: ")
+# factorial = 1
+
+
+# while 1 < num:
+#  factorial *= num
+#  num -= 1
+# print(factorial)
+
```
Comment (kolyasalubov) [inline VladislavPanchehin/HW5/5.3.py:11]: good job
but look at this implementation

n = int(input("Enter your number: "))


f = 1

# Checking if the entered value is an integer and non-negative
if isinstance(n, int) and n >= 0:
    # Calculating and printing factorials
    for item in range(2, n + 1):
        f *= item
        print(f"{item}! = {f}")

else:
    # Providing feedback if the entered number is not a non-negative integer
    print("Invalid input. Please enter a non-negative integer.")

**Review by kolyasalubov — APPROVED** [review]

File: VladislavPanchehin/HW5/5.3.py (hunk 3 above)
Comment (Vladislav-Panchehin) [reply VladislavPanchehin/HW5/5.3.py:11]: +

**Pull request merged**